package.json

```diff
@@ -5,6 +5,7 @@
   "devDependencies": {
     "babel": "^5.8.29",
     "babel-eslint": "4.1.3",
+    "babel-preset-react": "^6.0.15",    
```

Comment:
Which means that installing this preset should also be added to the inline instructions, am I correct? Babel 6.0 will not transpile the jsx file without the react preset as far as I can see.